HotSpot/CurrentParkingSpotsViewController.m

```diff
@@ -7,26 +7,55 @@
 //
 
 #import "CurrentParkingSpotsViewController.h"
+#import "SpotCell.h"
+
+@interface CurrentParkingSpotsViewController () <UITableViewDataSource, UITableViewDelegate>
```

Comment:
looks like this is identical to the BookedParkingSpotsViewController. can we reuse it? You can attach the same view controller to multiple Storyboard screens, if that's the concern